source/_posts/2016-05-28-display-part-of-rails-template-with-different-locale.markdown

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Show part of a Rails template in other language"
+date: 2016-05-30 10:06:00 +0200
+comments: true
+author: Ivica Lakatoš
+categories: 
+  - Ruby and Rails
+tags: 
+  - ruby
+  - rails
+  - I18n.locale
+published: true
+---
+
+I recently worked on a Rails project, which had parts of pages in different languages. That may be a problem if you have already translated their entire text to all required languages. You can even be tempted to hardcode parts of the text into other languages. Fortunately, there is an elegant way to solve that problem, just wrap parts of template or partials into blocks with different locale, like this:
```

Comment:
@Ikera  did you maybe mean 'wrap parts of template or partials into block with desired locale'?